ReactiveUI/ObservableAsPropertyHelper.cs

```diff
@@ -112,10 +114,17 @@ public sealed class ObservableAsPropertyHelper<T> : IHandleObservableErrors, IDi
         /// </summary>
         public IObservable<Exception> ThrownExceptions { get; private set; }
 
+        private void Activate()
+        {
+            if ( _activated )
```

Comment:
I would use Interlock together with a SingleAssignmentDisposable to guarantee both single subscription and disposal, keeping the previous logic unchanged:
```
                if (Interlocked.CompareExchange(ref _activated, 1, 0) == 0) {
                    _disp.Disposable = _source.Connect();
                }
```